### ISSUE TITLE
warning: Use of deprecated "Deno.stdout.rid" API. This API will be removed in Deno 2.

### ISSUE THREAD

**Issue by 2A5F** [deno-library/progress#26]
deno version: `1.40.0`
```
warning: Use of deprecated "Deno.stdout.rid" API. This API will be removed in Deno 2.

Stack trace:
  at ProgressBar.get ttyColumns (https://deno.land/x/progress@v1.4.4/mod.ts:236:34)
  at ProgressBar.render (https://deno.land/x/progress@v1.4.4/mod.ts:157:12)

hint: Use `Deno.stdout` instance methods instead.
hint: It appears this API is used by a remote dependency. Try upgrading to the latest version of that dependency.
```
```
warning: Use of deprecated "Deno.isatty()" API. This API will be removed in Deno 2.

Stack trace:
  at ProgressBar.get ttyColumns (https://deno.land/x/progress@v1.4.4/mod.ts:236:15)
  at ProgressBar.render (https://deno.land/x/progress@v1.4.4/mod.ts:157:12)

hint: Use `Deno.stdin.isTerminal()`, `Deno.stdout.isTerminal()` or `Deno.stderr.isTerminal()` instead.
hint: It appears this API is used by a remote dependency. Try upgrading to the latest version of that dependency.
```

**Comment by fuxingZhang** [comment]
> deno version: `1.40.0`
> 
> ```
> 
> warning: Use of deprecated "Deno.stdout.rid" API. This API will be removed in Deno 2.
> 
> 
> 
> Stack trace:
> 
>   at ProgressBar.get ttyColumns (https://deno.land/x/progress@v1.4.4/mod.ts:236:34)
> 
>   at ProgressBar.render (https://deno.land/x/progress@v1.4.4/mod.ts:157:12)
> 
> 
> 
> hint: Use `Deno.stdout` instance methods instead.
> 
> hint: It appears this API is used by a remote dependency. Try upgrading to the latest version of that dependency.
> 
> ```
> 
> ```
> 
> warning: Use of deprecated "Deno.isatty()" API. This API will be removed in Deno 2.
> 
> 
> 
> Stack trace:
> 
>   at ProgressBar.get ttyColumns (https://deno.land/x/progress@v1.4.4/mod.ts:236:15)
> 
>   at ProgressBar.render (https://deno.land/x/progress@v1.4.4/mod.ts:157:12)
> 
> 
> 
> hint: Use `Deno.stdin.isTerminal()`, `Deno.stdout.isTerminal()` or `Deno.stderr.isTerminal()` instead.
> 
> hint: It appears this API is used by a remote dependency. Try upgrading to the latest version of that dependency.
> 
> ```

can you submit a pull request ?

**Comment by ghost** [comment]
Same issue, cluttering my terminal. :(

**Comment by fuxingZhang** [comment]
Landed in [v1.4.5](https://github.com/deno-library/progress/releases/tag/v1.4.5)
